pages/users/[id].tsx

```diff
@@ -154,11 +159,12 @@ export default function User({ loginContext }: Props) {
   const avatarColor = graffitiToColor($user.graffiti)
   const ordinalRank = numberToOrdinal($user.rank)
   const startDate = new Date(2021, 11, 1)
+  const endDate = nextMondayFrom(nextMonday(new Date()))
```

Comment:
It took me awhile to understand why this was happening. It gets the next Monday in the local time zone, and then gets the next Monday from that in the UTC time zone, I imagine this it to work around the fact that something can be past UTC-0 weekly cut off so it adds an extra week for safety?

Reply:
Exactly; it's a little non-intuitive but it seems to work correctly